test/TabExpansion.Tests.ps1

```diff
@@ -137,16 +137,70 @@ Describe 'TabExpansion Tests' {
             }
         }
         It 'Tab completes non-ASCII file name' {
-            git config core.quotepath true # Problematic (default) config
+            git.exe config core.quotepath true # Problematic (default) config
 
             $fileName = "posh$([char]8226)git.txt"
-            New-Item $fileName
+            New-Item $fileName -ItemType File
 
-            $GitStatus = & $module Get-GitStatus
-
-            $result = & $module GitTabExpansionInternal 'git add ' $GitStatus
+            $gitStatus = & $module Get-GitStatus
 
+            $result = & $module GitTabExpansionInternal 'git add ' $gitStatus
             $result | Should BeExactly $fileName
         }
     }
+
+    Context 'PowerShell Special Chars Tests' {
+        BeforeAll {
```

Comment:
This is bound to be a pretty common pattern. Is it possible to write a Pester helper that would encapsulate a `BeforeAll`/`AfterAll` pairing that initializes and cleans up a Git repo, with its path left in `$repoPath`? Something like:

```powershell
Context 'Awesome test' {
    TempRepo {
        # Set up the repo
    }
    It 'Uses $repoPath' {
        # ...
    }
}
```